packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_IPOUHarmonicCoarseOperator_def.hpp

```diff
@@ -189,27 +182,21 @@ namespace FROSch {
     {
         FROSCH_DETAILTIMER_START_LEVELID(buildCoarseSpaceTime,"IPOUHarmonicCoarseOperator::buildCoarseSpace");
 
-        this->NumberOfBlocks_ = repeatedNodesMapVec.size();
+        UN TotalNumberOfBlocks = repeatedNodesMapVec.size();
```

Comment:
Why was it necessary to add this?

Reply:
I am incrementing ``this->NumberOfBlocks_`` inside ``if (useForCoarseSpace)``.